pkg/web/routes.go

```diff
@@ -150,6 +150,9 @@ func (s *Server) setupRoutes() (err error) {
 			transactions.POST("/prepare", authorize(permiss.TravelRuleManage), s.PrepareTransaction)
 			transactions.POST("/send", authorize(permiss.TravelRuleManage), s.SendPreparedTransaction)
 
+			// Export method to export transactions to a CSV
+			transactions.GET("/csv", authorize(permiss.TravelRuleManage), s.ExportTransactions)
```

Comment:
Would `/v1/transactions/export` be better … I'm kind of thinking it would be, any other ideas?

Reply:
That's fine and it's more consistent with other routes.